Package.resolved

```diff
@@ -66,11 +66,11 @@
       },
       {
         "package": "xcproj",
-        "repositoryURL": "https://github.com/xcodeswift/xcproj.git",
+        "repositoryURL": "https://github.com/rahul-malik/xcproj.git",
         "state": {
           "branch": null,
-          "revision": "fc8cbfce8a6bc1a98773181afe8da97c2516b722",
-          "version": "1.2.0"
+          "revision": "1923c0f53d9dfb7257b2ff08e4819be8efe25001",
```

Comment:
This file needs to be updated as well with a `swift package update`

Reply:
@yonaskolb - FYI - this command also updates `Yams` which I've omitted from this PR unless you'd like to update it. Also I think the `Package.swift` should utilize either revisions or `.exact` for version numbers since the current `Package.swift` at any given commit doesn't specify the exact versions of the dependencies it requires. This can cause issues if there are breaking changes in minor version updates.